src/user.clj

```diff
@@ -0,0 +1 @@
+(ns user)
```

Comment:
Remove file

Reply:
I know that this file is the wrong place, but shouldn't we keep a user namespace to work while developing?